Source/Upstream: Yes, fixed in git master
Reason: Fix build with ffmpeg 8

From 0a93f7078b2ab3a8c08bd5e995f21ca7b126f3e8 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Lluc=20Sim=C3=B3?= <lluc.simo@protonmail.com>
Date: Wed, 26 Feb 2025 23:00:09 +0100
Subject: [PATCH] [source_avcodec] stop using deprecated
 {in,out}_channel_layout options (closes gh-397)

---
 src/io/source_avcodec.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/io/source_avcodec.c b/src/io/source_avcodec.c
index 5ebaff16..4abe9f8b 100644
--- a/src/io/source_avcodec.c
+++ b/src/io/source_avcodec.c
@@ -347,8 +347,8 @@ void aubio_source_avcodec_reset_resampler(aubio_source_avcodec_t * s)
     av_channel_layout_default(&input_layout, s->input_channels);
     av_channel_layout_default(&output_layout, s->input_channels);
 
-    av_opt_set_chlayout(avr, "in_channel_layout",  &input_layout,        0);
-    av_opt_set_chlayout(avr, "out_channel_layout", &output_layout,       0);
+    av_opt_set_chlayout(avr, "in_chlayout",  &input_layout,        0);
+    av_opt_set_chlayout(avr, "out_chlayout", &output_layout,       0);
 #else
     int64_t input_layout = av_get_default_channel_layout(s->input_channels);
     int64_t output_layout = av_get_default_channel_layout(s->input_channels);
